Src/WitsmlExplorer.Frontend/models/wellbore.tsx

```diff
@@ -3,7 +3,7 @@ import BhaRun from "./bhaRun";
 import LogObject from "./logObject";
 import Measure from "./measure";
 import MessageObject from "./messageObject";
-import MudLogObject from "./mudLog";
+import mudLog from "./mudLog";
```

Comment:
All imports seem to be aliased with lowercase first letter, whilst they should be uppercase. Probably a search-replace oversight 😬

Reply:
Fixed